test/e2e/test_serve.py

```
@@ -556,13 +557,15 @@ def test_quadlet_and_kube_generation_with_container_registry(container_registry,
             container_spec = containers_spec[0]
             assert 'command' not in container_spec
             model_alias = f"{container_registry.host}:{container_registry.port}/{test_model.split(':')[0]}"
+            expected_host = container_spec['args'][container_spec['args'].index("--host") + 1]
+            assert expected_host in ["::", "0.0.0.0"]
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)